unit-test/deleteTask.js

```diff
@@ -0,0 +1,9 @@
+const deleteTask = (taskList, index) => {
+  const newTaskList = [...taskList];
+  if (index > -1 && index < newTaskList.length) {
+    newTaskList.splice(index, 1);
+  }
+  return newTaskList;
+};
```

Comment:
- [ ] The main purpose of writing unit tests is to make sure the real functions are tested. Kindly ensure not a create another function, rather test the main function responsible for deleting the tasks